ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -373,11 +388,10 @@ class PprofExporter(exporter.Exporter):
     ):
         # type: (...) -> _stack_event_group_key_T
 
-        if event.trace_type == ext.SpanTypes.WEB.value:
-            trace_resource = self._none_to_str(event.trace_resource)
-        else:
-            # Do not export trace_resource for privacy concerns.
-            trace_resource = ""
+        trace_resource = None
+        # Do not export trace_resource for non Web spans for privacy concerns.
+        if event.trace_resource_container and event.trace_type == ext.SpanTypes.WEB.value:
+            trace_resource = event.trace_resource_container[0]
```

Comment:
A Pythonic way to "unbox" this would be
```suggestion
            trace_resource, = event.trace_resource_container
```
just a suggestion, feel free to ignore